src/dp_port.c

```diff
@@ -506,7 +506,7 @@ static int dp_port_public_flow_meter_config(struct dp_port *port, uint64_t publi
 
 int dp_port_meter_config(struct dp_port *port, uint64_t total_flow_rate_cap, uint64_t public_flow_rate_cap)
 {
-	if (dp_conf_get_nic_type() == DP_CONF_NIC_TYPE_TAP)
```

Comment:
Maybe it would be better to switch the condition around to indicate that "this only works for 'hw'" option and is skipped for everything else.

Reply:
Everything now looks good. But maybe we should emit a warning here so that there is a log line about the metering not being in effect due to the NIC type?

Reply:
Done, would make sense. Maybe better no warning when metering values are are both zero ?

Reply:
Ah true, especially since calling this function with zeroes is used for rollback/disable of metering.